pl/reference/tutorial.rst

```diff
@@ -72,7 +72,7 @@ Zarządzanie zależnościami
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 Bardzo ważnym pojęciem, które musi być zrozumiane podczas pracy z Phalconem jest jego :doc:`dependency injection container <di>`. Może to brzmieć bardzo skomplikowanie, ale jest bardzo proste i praktyczne.
 
-A service container is a bag where we globally store the services that our application will use to function. Each time the framework requires a component, it will ask the container using an agreed upon name for the service. Since Phalcon is a highly decoupled framework, :doc:`Phalcon\\Di <../api/Phalcon_Di>` acts as glue facilitating the integration of the different components achieving their work together in a transparent manner.
+Kontener Zależności jest zbiorem, który globalnie przechowuje wszelkie serwisy, z których nasza aplikacja korzysta w danych funkcjach. Za każdym razem gdy Framework wymaga któregoś komponentu, zapyta Kontener o uzgodnioną nazwę danego serwisu. Od kiedy Phalcon jest wysoce rozczłonkowanym Frameworkiem, Kontener Zależności pełni rolę kleju, który łączy różne komponenty osiągając ich synergię.  
```

Comment:
`korzysta do funkcjonowania` maybe?
Also i don't like this Since translate too, well it's weird in first place. Maybe more like: `Z racji że Phalcon jest wysoce rozdzielnym Frameworkiem`? Don't like this `rozłocznkowanym` too.